source/administration/identity-access-management/oidc-access-management.rst

```diff
@@ -15,18 +15,64 @@ MinIO supports using an OpenID Connect (OIDC) compatible IDentity Provider (IDP)
 such as Okta, KeyCloak, Dex, Google, or Facebook for external management of user
 identities.
 
-For identities managed by the external OpenID Connect (OIDC) compatible provider, MinIO uses the `JSON Web Token claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4>`__ returned as part of the OIDC authentication flow to identify the :ref:`policies <minio-policy>` to assign to the authenticated user.
+For identities managed by the external OpenID Connect (OIDC) compatible provider, MinIO can use either of two methods to assign policies to the authenticated user.
+
+1. Use the `JSON Web Token claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4>`__ returned as part of the OIDC authentication flow to identify the :ref:`policies <minio-policy>` to assign to the authenticated user.
+2. Use the ``RoleARN`` specified in the authorization request to assign the policies attached to the provider's Role Policy.
+   
 
 MinIO by default denies access to all actions or resources not explicitly allowed by a user's assigned or inherited :ref:`policies <minio-policy>`. 
 Users managed by an OIDC provider must specify the necessary policies as part of the JWT claim. If the user JWT claim has no matching MinIO policies, that user has no permissions to access any action or resource on the MinIO deployment.
 
 The specific claim which MinIO looks for is configured as part of :ref:`deploying the cluster with OIDC identity management <minio-external-iam-oidc>`. This page focuses on creating MinIO policies to match the configured OIDC claims.
 
-Authentication and Authorization Flow
+Authentication and Authorization Flow 
 -------------------------------------
 
+A MinIO cluster can have any number of Role Policy based OpenID providers configured.
```

Comment:
Suggest we intro lead and do the micro-toc here

Spitballing:

```
MinIO supports two OIDC authentication and authorization flows:

1. The RolePolicy flow sets the assigned policies for an authenticated user in the MinIO configuration.

2. The JWT flow sets the assigned policies for an authenticated user as part of the OIDC configuration.

MinIO supports **one** JWT claim-based OIDC provider per deployment. All other providers must use Role Policy.

Reply:
Applied this suggestion with some small tweaks.